managedtenants/utils/ocm.py

```diff
@@ -104,67 +106,100 @@ def token(self):
         return self._token
 
     def list_addons(self):
-        return self._pool_items("/api/clusters_mgmt/v1/addons")
+        return self._pool_items(self.ADDON_MGMT_API_URL_PREFIX)
 
     def list_sku_rules(self):
         return self._pool_items("/api/accounts_mgmt/v1/sku_rules")
 
     def add_addon(self, metadata):
         addon = self._addon_from_metadata(metadata)
-        return self._post("/api/clusters_mgmt/v1/addons", json=addon)
+        return self._post(
+            self.ADDON_MGMT_API_URL_PREFIX, json=addon)
+
+    # Update Tooling to point to new addon-service API MTSRE-601
+    def add_addon_as(self, metadata):
+        addon = self._addon_from_metadata(metadata)
+        return self._post(
+            self.ADDON_MGMT_NEW_API_URL_PREFIX, json=addon)
 
     def _addon_exists(self, addon_id):
         try:
-            self._get(f"/api/clusters_mgmt/v1/addons/{addon_id}")
+            self._get(f"{self.ADDON_MGMT_API_URL_PREFIX}/{addon_id}")
             return True
         # `_get` raises OCMAPIError on 404's
         except OCMAPIError:
             return False
 
     def add_addon_version(self, imageset, metadata):
-        if self._addon_exists(metadata.get("id")):
-            return self._add_addon_version(imageset, metadata)
-        # Create the addon first
-        self.add_addon(metadata)
-        return self._add_addon_version(imageset, metadata)
+        # Create the addon first if it does not exisit
+        if self._addon_exists(metadata.get("id")) is False:
+            self.add_addon(metadata)
+        addon = self._addon_from_imageset(imageset, metadata)
+        return self._post(
+            f'''{self.ADDON_MGMT_API_URL_PREFIX}/
+            {metadata.get("id")}/versions''',
+            json=addon)
 
-    def _add_addon_version(self, imageset, metadata):
+
```

Comment:
Let's restore the `_add_addon_version` method so that we can still use it for CS-related methods like it was being used previously